### PR TITLE
Add nf-quilt 0.3.2 from quiltdata

### PR DESCRIPTION
**Pull request by drernie**

### CODE REVIEW

**Comment by drernie** [comment]
@pditommaso Let me know if you need anything else

File: plugins.json [hunk 1]
```diff
@@ -1282,6 +1282,13 @@
         "date": "2022-08-23T18:35:02.945086+02:00",
         "sha512sum": "1620b341b3a58167fb8f6a8d7f731be6ada7b77e224d8c434b378a3b319612619cf42f959fdeb007c96169d20d45fa85d6e47414478c441edc04f27b19668d25",
         "requires": ">=22.04.0"
+      },
+      {
+        "version": "0.3.0",
+        "url": "https://github.com/nextflow-io/nf-quilt/releases/download/0.3.0/nf-quilt-0.3.0.zip",
```
Comment (pditommaso) [inline plugins.json:1288]: Think this is the wrong repo. Likely you need to update your build script with your own org

Comment (drernie) [reply plugins.json]: Whoops, fixed manually for now. Will fix in the build script going forward.

Comment (drernie) [reply plugins.json]: The initial meta was corrupt due to a missing `gradle.properties`. Pushed a new 0.3.1 that has the correct URL, and added notes to the Makefile to detect and warn about the problem.

**Comment by drernie** [comment]
@pditommaso Any update? I believe I fixed the one issue you were concerned about. Is there anything else?

File: plugins.json [hunk 2]
```diff
@@ -1282,6 +1282,27 @@
...
+        "date": "2023-01-31T10:42:24.764088-08:00",
+        "sha512sum": "31f91a803dc0a10ac366190481e5bdba0af52e6d3fa0db7e540b86aacebe8da443dc1bfb28c8742e1181e7c6dc785944706ed7ee2fe1ae0da8729d84fbc248a5",
+        "requires": ">=23.01.0-SNAPSHOT"
+      },
+      {
+        "version": "0.3.1",
+        "url": "https://github.com/quiltdata/nf-quilt/releases/download/0.3.1/nf-quilt-0.3.1.zip",
+        "date": "2023-01-31T11:49:28.859425-08:00",
+        "sha512sum": "8518d236fde1cf440ce73efd2b65c505a3eeeac0fa94d40aab8b34878384b883107b6968a0e9880e2861338b97f9a07425f750b00b6ec831bff86ce2d39c58ad",
+        "requires": ">=23.01.0-SNAPSHOT"
```
Comment (drpatelh) [inline plugins.json:1298]:
```suggestion
        "requires": ">=22.10.6"
```

File: plugins.json [hunk 3]
```diff
@@ -1282,6 +1282,27 @@
         "date": "2022-08-23T18:35:02.945086+02:00",
         "sha512sum": "1620b341b3a58167fb8f6a8d7f731be6ada7b77e224d8c434b378a3b319612619cf42f959fdeb007c96169d20d45fa85d6e47414478c441edc04f27b19668d25",
         "requires": ">=22.04.0"
+      },
+      {
+        "version": "0.3.0",
+        "url": "https://github.com/quiltdata/nf-quilt/releases/download/0.3.0/nf-quilt-0.3.0.zip",
+        "date": "2023-01-31T10:42:24.764088-08:00",
+        "sha512sum": "31f91a803dc0a10ac366190481e5bdba0af52e6d3fa0db7e540b86aacebe8da443dc1bfb28c8742e1181e7c6dc785944706ed7ee2fe1ae0da8729d84fbc248a5",
+        "requires": ">=23.01.0-SNAPSHOT"
```
Comment (drpatelh) [inline plugins.json:1291]:
```suggestion
        "requires": ">=22.10.6"
```

**Comment by drernie** [comment]
@drpatelh removed the obsolete versions

**Comment by pditommaso** [comment]
Ok, let's go ahead with this. Sorry for the delay

**Pull request merged**